web/src/repo/settings/RepoSettingsCodeIntelligencePage.tsx

```diff
@@ -0,0 +1,125 @@
+import * as GQL from '../../../../shared/src/graphql/schema'
+import * as React from 'react'
+import { eventLogger } from '../../tracking/eventLogger'
+import { PageTitle } from '../../components/PageTitle'
+import { RouteComponentProps } from 'react-router'
+import { fetchLsifDumps } from './backend'
+import { FilteredConnection, FilteredConnectionQueryArgs } from '../../components/FilteredConnection'
+import { Timestamp } from '../../components/time/Timestamp'
+import { Link } from '../../../../shared/src/components/Link'
+import { catchError, map } from 'rxjs/operators'
+import { Observable } from 'rxjs'
+import { LoadingSpinner } from '@sourcegraph/react-loading-spinner'
+import { sortBy } from 'lodash'
+import { ErrorLike } from '@sourcegraph/codeintellify/lib/errors'
+import { isErrorLike } from '../../../../shared/src/util/errors'
+import { useObservable } from '../../util/useObservable'
+
+interface LsifDumpNodeProps {
+    node: GQL.ILSIFDump
+}
+
+class LsifDumpNode extends React.PureComponent<LsifDumpNodeProps, {}> {
+    public render(): JSX.Element | null {
+        return (
+            <div className="lsif-dump list-group-item">
```

Comment:
The root CSS class of the component should match the component name. See https://github.com/sourcegraph/sourcegraph/blob/master/doc/dev/web_app.md#styles
```suggestion
            <div className="lsif-dump-node list-group-item">
```